generator/src/main/java/com/scottlogic/deg/generator/Visualise.java

```diff
@@ -10,59 +9,66 @@
 import com.scottlogic.deg.generator.fieldspecs.FieldSpecMerger;
 import com.scottlogic.deg.generator.fieldspecs.RowSpecMerger;
 import com.scottlogic.deg.generator.validators.StaticContradictionDecisionTreeValidator;
+import picocli.CommandLine;
 
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStreamWriter;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Path;
-import java.util.List;
 import java.util.Objects;
-import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-@picocli.CommandLine.Command(
+@CommandLine.Command(
     name = "visualise",
     description = "Produces a decision tree in DOT format for the specified profile.",
-    mixinStandardHelpOptions = true,
-    version = "1.0")
+    descriptionHeading = "%nDescription:%n",
+    parameterListHeading = "%nParameters:%n",
+    optionListHeading = "%nOptions:%n",
+    abbreviateSynopsis = true)
 public class Visualise implements Runnable {
-    @picocli.CommandLine.Parameters(index = "0", description = "The path of the profile json file.")
-    private File sourceFile;
+    @CommandLine.Parameters(index = "0", description = "The path of the profile json file.")
+    private File profileFile;
 
-    @picocli.CommandLine.Parameters(index = "1", description = "The directory into which generated data should be saved.")
-    private Path outputDir;
+    @CommandLine.Parameters(index = "1", description = "The path of the output visualise file.")
+    private File outputFile;
```

Comment:
This has actually been changed (not just the wording) to take a file instead of a directory path for the output target as part of this PR to make it more in line with generate

Reply:
You _could_ leave it as a `Path` which would mean that you wouldn't need to call `toPath()` later on. Equally `File` describes clearly that it must be a _file_ and not a general _path_.

What happens if you run the command with a directory path, can Pico resolve the directory path to a `File`?

Reply:
with these changes it will not be able to resolve a directory path to a file, shall we revert?

Reply:
I'd say change it back to a `Path`?